### PR TITLE
Launch Linux browser windows in different sessions

### PR DESCRIPTION
**Pull request by hlascelles**
If `google-chrome` is present, launch a new session window for each invocation.

### CODE REVIEW

File: zsh-aws-vault.plugin.zsh [hunk 1]
```diff
@@ -48,6 +48,14 @@ function avli() {
         ;;
     esac
 
+  elif (command -v "google-chrome" > /dev/null); then
```
Comment (blimmer) [inline zsh-aws-vault.plugin.zsh]: I'd love to see this block follow the pattern above, e.g.

```
elif _using_linux ; then
  case $browser in
     google-chrome)
         #your logic
         ;;
   esac
```

That should be pretty straightforward to do, by adding a `uname` check for `Linux` and adjusting the `_find_browser` to have logic for Linux (using the `command -v` logic you have would be perfect). It's totally fine if you only implement the `google-chrome` logic and leave the rest for others who are willing to PR the project for Firefox, etc.

**Review by blimmer — CHANGES_REQUESTED** [review]
Hi @hlascelles, and thank you for taking the time to create this PR. Since this is the first logic explicitly intended for Linux, I added an inline comment asking if you could create the skeleton for other Linux logic in the future.

If you have the time to make those changes, that would be great! If not, let me know, and I can try to find some time to add that and integrate your logic. Thanks!

**Comment by hlascelles** [comment]
Sure thing, I will look into it :+1:

**Comment by hlascelles** [comment]
So, turns out that much of the Mac line also worked. On Linux it makes sense to maximise it too though.

How does it look?

**Review by blimmer — APPROVED** [review]
This looks great! Thank you for making the requested changed :-)

**Pull request merged**